content/influxdb/cloud-iox/reference/sql/information-schema.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Information schema
+description: > 
+    ...
+menu:
+  influxdb_cloud_iox:
+    parent: SQL reference
+weight: 210
+---
+
+The underlying query engine for the InfluxDB SQL implementation,
+[DataFusion](https://arrow.apache.org/datafusion/index.html), provides commands
+that return metadata related to your data schema.
+To access this information, use the the `SHOW TABLES`, `SHOW COLUMNS`, and
+`SHOW ALL` commands or query views of in [ISO](https://www.iso.org/) SQL
```

Comment:
```
`SHOW ALL` commands or query views of in [ISO](https://www.iso.org/) SQL
```

I'm not sure what this is supposed be, but seems like a typo?

Reply:
```suggestion
`SHOW ALL` commands or query views in the [ISO](https://www.iso.org/) SQL
```